File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    var answer = [];
    for(var i = 0; i < arr.length; i++) {
        if(arr[i] % divisor == 0) answer.push(arr[i]);
    }
    return answer.length ? answer.sort((a, b) => a - b) : [-1];
}

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        ArrayList<Integer> list = new ArrayList<>();
        for(int i = 0; i < arr.length; i++) {
            if(arr[i] % divisor == 0) list.add(arr[i]);
        }
        Collections.sort(list);
        int[] answer = new int[list.size()];
        for(int i = 0; i < list.size(); i++) {
            answer[i] = list.get(i);
        }
        return answer.length == 0 ? new int[]{-1} : answer;
    }
}